src/test/java/org/junitpioneer/testkit/PioneerTestKit.java

```diff
@@ -38,12 +41,21 @@ public static ExecutionResults executeTestMethod(Class<?> testClass, String test
 	 *
 	 * @param testClass Name of the test class
 	 * @param testMethodName Name of the test method (of the given class)
-	 * @param methodParameterTypes Full qualified types names of the parameters (e.g. "java.nio.file.Path")
+	 * @param methodParameterTypes Class type(s) of for the parameters
```

Comment:
It's either `of` or `for` but never both.

Reply:
Changed, thanks